doc/en/_templates/sidebarintro.html

```diff
@@ -1,5 +1,8 @@
 <h3>About pytest</h3>
 <p>
-  pytest is a mature full-featured Python testing tool that helps
+  PyTest is a mature full-featured Python testing tool that helps
   you write better programs.
 </p>
+<p>
+  you can write code to test anything like database,API,even UI if you want.
```

Comment:
Hi @readthroughmyglasses,

Unfortunately I don't see how this improves the blurb on the side bar. Ideally it should stay short (1 paragraph), and concise. Plus you have a few grammar mistakes: `you` instead of `You`, and missing spaces between commas.